events/events_test.go

```diff
@@ -137,19 +147,19 @@ func TestFanout1(t *testing.T) {
 							break listenLoop
 						} else {
 							if sdata.x != x+1 {
-								log.Fatalf("Did not get correct data in event.\n")
+								log.Fatalf("Did not get correct data in event.")
 							}
 							x = sdata.x
 						}
 					} else {
-						log.Fatalf("Failed type assertion\n")
+						log.Fatalf("Failed type assertion")
 					}
 					latchon(hook)
 					// default:
 					//     latchon(hook)
 				}
 			} else {
-				log.Fatalf("channel should be available. wrong.\n")
+				log.Fatalf("channel should be available. wrong.")
```

Comment:
Please scan and merge your changes to master -

newline is required for `format` string API's implemented with `printf` `sprintf` etc - usually ending with `f` and not needed for API's implemented using `Println`